LICENSE.txt

```diff
@@ -0,0 +1,14 @@
+Copyright (c) 2017+ LocalStack contributors
+Copyright (c) 2016 Atlassian Pty Ltd
```

Comment:
```suggestion
Copyright (c) 2023 LocalStack
```

Reply:
Started a discussion with @tinyg210 if we should add the entire license text or not (still being decided). 
However, we should add the copyright for `2023 LocalStack` (I am not sure why in the linked version Atlassian was mentioned as well).

Reply:
the only thing which we should be caring about is having the license discoverable by github, which i checked, is happening with the short version. so from my side the short one is suitable. we still need to adapt it to use our company only though.

Reply:
Thanks for catching this, that was my bad, I simply copy pasted the LocalStack one without really checking, and there it still states Atlassian